### DISCUSSION TITLE
list-public-ip-addresses-of-ec2-instances-using-aws-cli

### DISCUSSION THREAD

**Discussion by giscus** [thomasnotfound/how.wtf#155] [Blog Posts]
# list-public-ip-addresses-of-ec2-instances-using-aws-cli

How to use list public IP addresses of EC2 instances using the AWS CLI

https://how.wtf/list-public-ip-addresses-of-ec2-instances-using-aws-cli.html

**Comment by ypark9** [comment]
Hello sir 🦘, Do you intend to only show the first tag of a post on the `url/articles.html` page? Because currently, when a post has both tag1 and tag2, only tag1 is displayed.
